packages/react-scripts/config/InjectEntrypointsPlugin.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+const { getCompilerHooks } = require('webpack-manifest-plugin');
+const fs = require('fs');
+const path = require('path');
+
+class InjectEntrypointsPlugin {
+  constructor(options) {
+    this.options = options;
+  }
+
+  apply(compiler) {
+	const {afterEmit} = getCompilerHooks(compiler)
+
+	afterEmit.tap('InjectEntrypointsPlugin', (manifest) => {
+		const outputPath = path.join(
+			compiler.options.context,
+			this.options.outputFile
+		);
+
+		fs.writeFileSync(outputPath,
+			JSON.stringify(
+				manifest.entrypoints.map((entry) => this.options.outputPath + entry)),
```

Comment:
The entrypoints didn't have a full path (prepended w/the CDN when doing a prod build) So I needed to do this to know where to look for them. However, I am going to change this variable name because it's confusing with line 15. And it's just `paths.publicUrlOrPath`so it doesn't even need passed in